### ISSUE TITLE
[Bug]: File lock prevents serverconfig modification

### ISSUE THREAD

**Issue by WrestlingNerd** [FTBTeam/FTB-Modpack-Issues#368]
### Modpack

FTB Oceanblock

### Modpack version

1.9.0

### Log Files

https://paste.feed-the-beast.com/view/df575930

### Describe the bug

When attempting to load onto a server or a single player world, it crashes and gives this error.

### Steps to reproduce

1. Load game perfectly
2. Attempt to load into server
3. Waiting on screen that says loading world. Would like to mention that it does load me into the server properly.
4. Crash

### Expected behaviour

Game crashes when loading into any world

**Comment by Raidobw2** [comment]
I see this error in the log:
`java.nio.file.FileSystemException: %localappdata%\.ftba\instances\11adb4b1-906a-44a5-b1ec-551cea307105\saves\New World\serverconfig\notenoughwands-server.toml: The process cannot access the file because it is being used by another process.`
Try restarting the FTB App or the launcher you're using. Please close the issue if that solves it.

**Comment by WrestlingNerd** [comment]
Have tried this several times.

**Comment by Raidobw2** [comment]
Alright, you can try something else:
- Close the FTB App.
- Delete the contents of `saves/world/serverconfig` for the problematic world(s).
- Open the FTB App.
- Load modpack and try to play.

If the error persists, check to see if you have programs that could cause file locks, such as an antivirus. You could disable the antivirus before creating the world, if it's the one causing the file(s) to lock.

Otherwise you can create a new custom instance on CurseForge and copy the files over. I'd recommend you to check https://github.com/FTBTeam/FTB-App-Feedback for any issues like yours, or create a new ticket so that our devs can try and fix the problem.
